api_app/serializers/job.py

```diff
@@ -500,6 +500,18 @@ def save(self, parent: Job = None, **kwargs):
         analysis.save()
         return result
 
+    def validate(self, attrs: dict) -> dict:
+        # filter requests with more elements than this threshold
+        max_element_per_request_number = 200
+        if len(attrs) > max_element_per_request_number:
+            raise ValidationError(
+                {
+                    "detail": "Exceed the threshold of "
+                    f"{max_element_per_request_number}  elements for a single analysis"
+                }
+            )
```

Comment:
In the validation error just put the message, not a dict

Reply:
If i put just the str the backend return:
```
{
    "errors": {
        "non_field_errors": [
            "Exceed the threshold of 2  elements for a single analysis"
        ]
    }
}
```
instead of:
```
{
    "errors": {
        "detail": [
            "Exceed the threshold of 2  elements for a single analysis"
        ]
    }
}
```

I can handle this in the frontend, but i'm not sure is better. I agree with you it will be better in the backend